### ISSUE TITLE
Améliorer la mise en forme des articles

### ISSUE THREAD

**Issue by barbogogo** [barbogogo/leedReader#2]
Dans les articles, mettre un lien vers l'article original, la date de publication, l'auteur.

**Comment by ghost** [comment]
Ce serait sympa d'avoir l'affichage de la première photo d'un article et un résumé en dessous du titre.
J'ai noté que la taille de police est un peu petite, c'est possible d'ajouter une option dans les paramètres pour ajuster la taille à sa convenance?

**Comment by barbogogo** [comment]
Salut,
Oui, j'y pense, mais je n'ai pas complètement terminé ma réflexion. D'ailleurs, cela peut être un sujet de discussion intéressant, d'où la réouverture de ce ticket :smile: !
En effet, en travaillant un peu sur le sujet, l'une des solutions retenues est de récupérer la première image trouvée dans l'article avec quelque chose du genre : `<img ([^><]*) src="([^="><]*)" ([^><]*)>`, où la seconde chaîne retournée par cette expression régulière est le chemin de l'image.
Toutefois, en faisant quelques tests, je me suis rendu compte que certains articles ne proposaient pas d'images... c'est à dire, soit l'expression régulière ne renvoie rien, soit (et c'est plus embêtant pour le traitement), elle renvoie une image de réseau social (du type le logo de google+ ou facebook, ...) qui sont des images liées aux liens de partages de l'article (et qui n'ont rien à voir avec le contenu de l'article). Bref, il faut que j'approfondisse la chose...

**Comment by ghost** [comment]
il faudrait soit que tu prévois une images aléatoire de remplacement s'il n'y a rien, soit l'espace dédié à l'image qui se libère pour laisser plus de place au résumé...
Et il faudrait des régles d'exlusions pour les images de certains domaines comme les réseaux sociaux. Feedly utilise des images de remplacement il me semble, mais surtout arrive à supprimer les images des réseaux sociaux.

**Comment by barbogogo** [comment]
Ok, je vais regarder.
Si il y a une image, peut-être qu'un test sur la taille de celle-ci est suffisant pour vérifier s'il s'agit d'une icône ou non...

**Comment by fran6t** [comment]
Moi je pense qu'il faudrait que ce soit Leed qui fasse cela, je m'explique ce traitement est utile aussi dans ou pour un futur thème leed qui permettrait d'afficher sous forme magazine. L'appli LeedReader pourrait récuperer donc déjà quelque chose qui a été travaillé au moment de la mises à jour du flux.

**Comment by barbogogo** [comment]
Pourqoui pas. 
@cobalt74, @sbgodin, qu'en pensez-vous?

**Comment by Sbgodin** [comment]
Je suis d'accord sur le principe aussi. Cependant, faut-il conserver le verbatim malgré tout et appliquer à la volée les changements ou changer définitivement dès réception ?
Autre question, tous les utilisateurs souhaitent-ils d'avoir ces modifications ? Si oui, veulent-ils tous les mêmes modifications ?
Question supplémentaire, dans le cas d'un hébergement multi-utilisateurs. Si la modification est faite au téléchargement, la configuration éventuelle s'imposera à tous.

De tout ça, je pense qu'il faudrait conserver le flux tel quel en base. Un série de filtre pourra être appliqué, voire même calculé par défaut et présenté par défaut. Libre à l'utilisateur de le configurer à sa guise. Il y aura donc le flux avec ses articles d'origine et les articles filtrés.

**Comment by cobalt74** [comment]
leedReader à son propre mode d'affichage (thème) et les thèmes sont sur la partie web uniquement.
l'API permet de récupérer toutes les informations qu'il faudrait pour faire les affichages nécessaires.
Rien n'empèche Leed ou leedReader d'avoir un thème magazine qui gère les images en présentation d'article. :)

**Comment by qwertygc** [comment]
![screenshot_2014-07-05-14-06-19](https://cloud.githubusercontent.com/assets/4261986/3486489/316acf5e-043d-11e4-825e-0889b2100cf6.png)
Bonjour !
Dans l’accueil, on voit le texte tronqué et il faut cliquer pour le lire. Est-ce possible d'avoir le texte en entier afin d'éviter de cliquer à tout va ?
Cordialement,
Qwerty

**Comment by barbogogo** [comment]
Bonjour,
En effet le texte est tronqué pour balayer les articles plus facilement. Toutefois un paramètre pourrait etre ajouté pour définir le nombre de caractères souhaités où 0 serait tout l'article... qu'en pensez-vous ?

**Comment by qwertygc** [comment]
Oui, merci beaucoup :)
